core/src/test/resources/org/apache/calcite/test/TopDownOptTest.xml

```diff
@@ -606,7 +606,7 @@ on r.job=s.job and r.ename=s.ename]]>
     <Resource name="planBefore">
       <![CDATA[
 LogicalProject(ENAME=[$0], JOB=[$1], SAL=[$2], COMM=[$3], ENAME0=[$4], JOB0=[$5], MGR=[$6], EXPR$3=[$7])
-  LogicalJoin(condition=[AND(=($1, $5), =($0, $4))], joinType=[inner])
+  LogicalJoin(condition=[AND(=($5, $1), =($4, $0))], joinType=[inner])
```

Comment:
This change looks odd. Is there a reason why `$5` is placed before `$1`?

Reply:
This is just the original plan, without any operands reordering, it is changed in CALCITE-3786 and revert back in this PR.